`semver-utils` isn't exported

- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)

`semver-utils` is imported in [`build/index.d.ts`](https://cdn.jsdelivr.net/npm/npm-check-updates@17.1.3/build/index.d.ts). But the library `semver-utils` isn't in the [`package.json`](https://cdn.jsdelivr.net/npm/npm-check-updates@17.1.3/package.json) `dependencies`.

---

## Steps to Reproduce

Files:

- `package.json`

  ```json
  {
      "name": "testcase",
      "version": "1.0.0",
      "type": "module",
      "dependencies": {
          "npm-check-updates": "17.1.3",
          "typescript": "5.6.2"
      }
  }
  ```

- `index.ts`

  ```typescript
  import ncu from "npm-check-updates";

  console.log(await ncu.run());
  ```

Steps:

1. `npm install`
2. `npx tsc --module nodenext index.ts`

## Current Behavior

<!-- Describe the existing (incorrect) behavior. -->
```
node_modules/npm-check-updates/build/index.d.ts:1:24 - error TS2307: Cannot find module 'semver-utils' or its corresponding type declarations.

1 import { SemVer } from 'semver-utils';
                         ~~~~~~~~~~~~~~


Found 1 error in node_modules/npm-check-updates/build/index.d.ts:1
```

## Expected Behavior

No error.

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -91,7 +91,13 @@ export default defineConfig(({ mode }) => ({
     dts({
       entryRoot: 'src',
       include: ['src'],
-      bundleTypes: true,
+      /**
+       * Inline semver-utils types so the emitted .d.ts has no external import.
+       * semver-utils is bundled into the JS, not a runtime dependency.
+       */
+      bundleTypes: {
+        bundledPackages: ['semver-utils', '@types/semver-utils'],
+      },
       insertTypesEntry: true,
       outDirs: 'build',
     }),

